playwright/menu-event-types.test.ts

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from "@playwright/test";
+
+// Using logged in state from globalSteup
+test.use({ storageState: "playwright/artifacts/proStorageState.json" });
+
+test("test menu event types", async ({ page }) => {
+  // Try to go homepage
+  await page.goto("/");
+  // It should redirect you to the event-types page
+  await page.waitForSelector("[data-testid=event-types]");
+  await page.goto('/event-types');
```

Comment:
Same as above. Also, we should use this opportunity to start doing visual regression tests for mobile and desktop. I think the menu is one good example for this. Examples:

- The bottom nav should be visible on mobile
- The side nav should be visible on desktop
- Every single menu items should be accesible/visible in each breakpoint
- Etc.